api_docs/playlists.md

```diff
@@ -185,3 +185,105 @@ video_id[]  | A comma separated list of video IDs to remove from the playlist |
   }
 }
 </pre>
+
+## Smart Ordering
+
+<p>Playlist smart ordering allow you to change the order of its videos by a sort options. The videos added or updated into a playlist with sort options, are sort automatically following that sort option</p>
```

Comment:
"Playlist smart ordering allows you" (allows)
"its videos by sort options" (remove 'a')
"The videos added or updated into a playlist with sort options are sorted automatically" (remove comma, sorted)